tensorboard/webapp/metrics/store/metrics_selectors_test.ts

```diff
@@ -425,45 +426,172 @@ describe('metrics selectors', () => {
   });
 
   describe('getMetricsCardTimeSelection', () => {
-    it('returns cards timeSelection if defined', () => {
-      const state = appStateFromMetricsState(
-        buildMetricsState({
+    describe('when linked time is disabled', () => {
+      let partialState: Partial<MetricsState>;
+      beforeEach(() => {
+        partialState = {
           linkedTimeEnabled: false,
-          cardStateMap: {
-            card1: {
-              timeSelection: {
-                start: {step: 0},
-                end: {step: 5},
+        };
+      });
```

Comment:
My intention here is to show an interconnectedness within these tests. Maybe just putting them all inside the `describe` block already does that and this shared `partialState` is just confusing?